test/test_product_config.py

```diff
@@ -226,6 +227,7 @@ def test_from_config_bool(self) -> None:
         assert options.develop.any_gpu is True
         assert options.develop.disable_ibverbs is True
         assert options.develop.less_resources is True
+        assert options.develop.data_timeout == 30.0
```

Comment:
```suggestion
        assert options.develop.data_timeout == defaults.RX_DEVICE_STATUS_TIMEOUT
```
to avoid repeating the magic number.

Reply:
Fair point.